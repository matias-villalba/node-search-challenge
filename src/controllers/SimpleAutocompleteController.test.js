const SimpleAutocompleteController = require('./SimpleAutocompleteController');

describe('SimpleAutocompleteController', () => {
  it('should return all results from suggestions', () => {
    const response = SimpleAutocompleteController.index('biology');

    expect(response.result.length).toEqual(29);
    expect(response.result).toMatchSnapshot();

  });

  it('should return no results when searchTerm is empty', () => {
    const response = SimpleAutocompleteController.index('');
    expect(response.result.length).toEqual(0);

  });

  it('should return no results when searchTerm is undefined', () => {
    const response = SimpleAutocompleteController.index();
    expect(response.result.length).toEqual(0);

  });

});
