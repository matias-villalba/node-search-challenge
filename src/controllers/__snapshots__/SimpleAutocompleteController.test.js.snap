// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleAutocompleteController should return all results from suggestions 1`] = `
Array [
  Object {
    "attribution": "Leboffe et al",
    "briefDescription": "",
    "chapterPrice": 1.99,
    "copyrightYear": 2011,
    "description": "Start prepping for the lab with the Inkling version of Leboffe's A Photographic Atlas for the Microbiology Laboratory. Zoom in on hundreds of hi-res stains, illustrations, charts and photos, curate your own study guide right in the atlas, and quickly find what you need with our powerful search that pulls results from the text, your notes, and even the web.
",
    "edition": "4th edition",
    "price": 34.99,
    "primaryCategory": "Science",
    "publisher": "Morton Publishing",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-15.jpeg",
    "title": "A Photographic Atlas for the Microbiology Laboratory",
    "url": "https://www.inkling.com/store/book/photographic-atlas-microbiology-laboratory-michael-leboffe-4th/",
  },
  Object {
    "attribution": "Raven et al",
    "briefDescription": "",
    "chapterPrice": 3.99,
    "copyrightYear": 2010,
    "description": "Need Raven Biology for your class? Try the Inkling version! Breeze through homework with interactive quizzes, visual guided tours, and quick reference tools like an interactive glossary. It’s everything you need for your course, evolved. No book required.",
    "edition": "9th edition",
    "price": 139.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-13.jpeg",
    "title": "Biology",
    "url": "https://www.inkling.com/store/book/biology-peter-raven-9th/",
  },
  Object {
    "attribution": "Sylvia S. Mader",
    "briefDescription": "",
    "chapterPrice": 4.99,
    "copyrightYear": 2009,
    "description": "Need Mader Biology for your class? Try the Inkling version! Breeze through homework with interactive quizzes, visual guided tours, and quick reference tools like an interactive glossary. It's everything from the paper version of Mader Biology... evolved.",
    "edition": "10th edition",
    "price": 139.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-04.jpeg",
    "title": "Biology",
    "url": "https://www.inkling.com/store/book/biology-sylvia-mader-10th/",
  },
  Object {
    "attribution": "Brooker et al ",
    "briefDescription": "",
    "chapterPrice": 3.99,
    "copyrightYear": 2010,
    "description": "Go behind the scenes of creatures great and small with Brooker Biology for iPad and the Web. Inkling's version is packed with interactive content: more than 40 3D molecules to spin, 200 videos to watch, and 350 enhanced figures to explore. Evolve what you know, with concept checks and customized quizzes. ",
    "edition": "2nd edition",
    "price": 129.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-03.jpeg",
    "title": "Biology",
    "url": "https://www.inkling.com/store/book/biology-robert-brooker-2nd/",
  },
  Object {
    "attribution": "Mariëlle Hoefnagels",
    "briefDescription": "",
    "chapterPrice": 4.99,
    "copyrightYear": 2011,
    "description": "Want to see what you're made of? Get Hoefnagel's Biology on Inkling. Listen to one of 100 audio tours by the author herself. You can also easily navigate your book with hundreds of cross-links between different chapters, figures, and more. Want to dive beneath the surface? Use one of the 100 Test Yourself figures and Guided Tours. Grasping the complexity of life on earth just got easier. ",
    "edition": "2nd edition",
    "price": 119.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "Concepts and Investigations",
    "thumbnail": "data/thumbnails/book-07.jpeg",
    "title": "Biology",
    "url": "https://www.inkling.com/store/book/biology-marielle-hoefnagels-2nd/",
  },
  Object {
    "attribution": "Sylvia Mader and Michael Windelspecht",
    "briefDescription": "",
    "chapterPrice": 4.99,
    "copyrightYear": 2012,
    "description": "Need Mader Biology for class? Try the Inkling version of the 11th edition! Breeze through homework with interactive quizzes, visual guided tours, and quick reference tools like an interactive glossary. It's everything from the paper version of Mader Biology...evolved.",
    "edition": "11th edition",
    "price": 149.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-09.jpeg",
    "title": "Biology",
    "url": "https://www.inkling.com/store/book/biology-sylvia-mader-michael-windelspecht-11th/",
  },
  Object {
    "attribution": "Rene Fester Kratz and Donna Rae Siegfried",
    "briefDescription": "",
    "chapterPrice": 1.99,
    "copyrightYear": 2010,
    "description": "Want to see what you're made of? Get Biology For Dummies on Inkling. Whether on iPad, iPhone or the web, easily navigate your book with hundreds of live cross-links with the book and to the web. Dive beneath the surface with Test Yourself figures, Guided Tours, and Slideshows. There's even a powerful search engine for finding just what you need. Grasping the complexity of life on earth just got easier.",
    "edition": "2nd edition",
    "price": 14.99,
    "primaryCategory": "Science",
    "publisher": "John Wiley & Sons, Inc",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-10.jpeg",
    "title": "Biology For Dummies",
    "url": "https://www.inkling.com/store/book/biology-dummies-kratz-siegfried-2nd/",
  },
  Object {
    "attribution": "Michael Lieberman PhD, Rick Ricer MD",
    "briefDescription": "",
    "chapterPrice": 5.99,
    "copyrightYear": 2013,
    "description": "Studying biochemistry, molecular biology, and genetics? Try the Inkling version! On Inkling, Lieberman and Ricer's BRS Biochemistry, Molecular Biology, and Genetics delivers critical concepts and clinical correlates with interactive slideshows and diagrams that let you test yourself. Plus, take a quiz of USMLE-style review questions with answer feedback so you'll ace your exams, no sweat.",
    "edition": "6th edition",
    "price": 46.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-01.jpeg",
    "title": "BRS Biochemistry, Molecular Biology, and Genetics",
    "url": "https://www.inkling.com/store/book/brs-biochemistry-molecular-biology-genetics-lieberman-6th/",
  },
  Object {
    "attribution": "Gartner et al",
    "briefDescription": "",
    "chapterPrice": 2.99,
    "copyrightYear": 2010,
    "description": "Master the concepts of cell biology and histology with this BRS title by Gartner et al. Interactive Test Yourself figures and over 1100 USMLE®-style questions make memorization a breeze. Best of all, right answer feedback includes links back to the text for further review. Ready to ace the boards? Start with the free chapter below.",
    "edition": "6th edition",
    "price": 39.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [
      "Science",
    ],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-03.jpeg",
    "title": "BRS Cell Biology and Histology",
    "url": "https://www.inkling.com/store/book/brs-cell-biology-and-histology-gartner-6th/",
  },
  Object {
    "attribution": "Louise Hawley et al.",
    "briefDescription": "",
    "chapterPrice": 5.99,
    "copyrightYear": 2013,
    "description": "Studying microbiology and immunology? Try the Inkling version! On Inkling, BRS Microbiology & Immunology delivers critical concepts in immunological function and human infection with interactive slideshows and diagrams that let you test yourself. Plus, take a quiz of USMLE-style review questions with answer feedback so you'll ace your exams, no sweat.",
    "edition": "6th edition",
    "price": 46.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-01.jpeg",
    "title": "BRS Microbiology & Immunology",
    "url": "https://www.inkling.com/store/book/brs-microbiology-immunology-hawley-6th/",
  },
  Object {
    "attribution": "Reece et al",
    "briefDescription": "",
    "chapterPrice": 3.99,
    "copyrightYear": 2010,
    "description": "Wish your biology book was as rich as the world it describes? On Inkling, Campbell Biology brings biology to life with hundreds of customized enhancements like 48 3D molecules, over 500 interactive images, and embedded animations. When you need to prep for the exam, use multiple study tools like quizzes and concept checks built into every section. ",
    "edition": "9th edition",
    "price": 129.99,
    "primaryCategory": "Science",
    "publisher": "Pearson",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-11.jpeg",
    "title": "Campbell Biology",
    "url": "https://www.inkling.com/store/book/biology-neil-campbell-and-jane-reece-9th/",
  },
  Object {
    "attribution": "Jeffrey K. Actor",
    "briefDescription": null,
    "chapterPrice": 3.99,
    "copyrightYear": 2012,
    "description": "Effectively merge basic science and clinical skills with Elsevier’s Integrated Review of Immunology and Microbiology. This title, part of the Integrated Review Series focuses on the core knowledge in immunology and microbiology while linking that information to related concepts from other basic science disciplines.   This concise and user-friendly reference provides crucial guidance for the early years of medical training and USMLE preparation.",
    "edition": "2nd edition",
    "price": 42.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "with Student Consult",
    "thumbnail": "data/thumbnails/book-12.jpeg",
    "title": "Elsevier's Integrated Review Immunology and Microbiology",
    "url": "https://www.inkling.com/store/book/elseviers-integrated-review-immunology-and-microbiology-actor-2nd/",
  },
  Object {
    "attribution": "Alberts et al",
    "briefDescription": "",
    "chapterPrice": 9.99,
    "copyrightYear": 2009,
    "description": "Need Essential Cell Biology for class? On Inkling, hundreds of the book's high-res illustrations have been custom-built as interactive diagrams. Tap on boxed Hot Spots to dive deep into various organelles, analyze processes at your own pace with 56 different Slidelines, and view more than 100 videos of cellular processes, as if you were looking through a microscope.
",
    "edition": "3rd edition",
    "price": 129.99,
    "primaryCategory": "Medicine",
    "publisher": "Garland Science",
    "subCategories": Array [
      "Science",
    ],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-11.jpeg",
    "title": "Essential Cell Biology",
    "url": "https://www.inkling.com/store/book/essential-cell-biology-bruce-alberts-3rd/",
  },
  Object {
    "attribution": "Kathleen Park Talaro and Barry Chess",
    "briefDescription": "",
    "chapterPrice": 5.99,
    "copyrightYear": 2012,
    "description": "Get to know the smallest of organisms up close and personal with Foundations in Microbiology by Talaro and Chess on Inkling. Tap key terms for instant definitions, zoom in on hundreds of hi-res images, take notes and share them with your friends, and find just what you need with a powerful search. Try the free chapter below to see for yourself!
",
    "edition": "8th edition",
    "price": 99.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-08.jpeg",
    "title": "Foundations in Microbiology",
    "url": "https://www.inkling.com/store/book/foundations-in-microbiology-talaro-chess-8th/",
  },
  Object {
    "attribution": "Abraham L. Kierszenbaum and Laura Tres",
    "briefDescription": "",
    "chapterPrice": 3.99,
    "copyrightYear": 2012,
    "description": "Need Kierszenbaum and Tres' Histology and Cell Biology by for class? The Inkling version of this text includes more than 50 interactive guided tours and 180 Test Yourself diagrams that let you test what you've learned. Plus, our powerful search will find just what you need in your notes, the text, and the web with ease. Download the free chapter now and see for yourself! ",
    "edition": "3rd edition",
    "price": 79.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [
      "Science",
    ],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-04.jpeg",
    "title": "Histology and Cell Biology",
    "url": "https://www.inkling.com/store/book/histology-cell-biology-kierszenbaum-tres-3rd/",
  },
  Object {
    "attribution": "Sylvia S. Mader and Michael Windelspecht",
    "briefDescription": "",
    "chapterPrice": 7.99,
    "copyrightYear": 2011,
    "description": "Need Mader and Windelspecht's Human Biology for class? The Inkling edition includes over 50 Test Yourself diagrams and more than 170 videos, animations and audio clips that explain complex concepts and processes. At the end of each chapter, take quizzes with answer feedback that include links to relevant areas of the text for further review.

",
    "edition": "12th edition",
    "price": 109.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-06.jpeg",
    "title": "Human Biology",
    "url": "https://www.inkling.com/store/book/human-biology-sylvia-mader-12th/",
  },
  Object {
    "attribution": "Bruce Carlson",
    "briefDescription": "",
    "chapterPrice": 5.99,
    "copyrightYear": 2014,
    "description": "Master the concepts you need to know with Human Embryology and Developmental Biology. Dr. Bruce M. Carlson's clear explanations provide an easy-to-follow \\"road map\\" through the most up-to-date scientific knowledge, giving you a deeper understanding of the key information you need to know for your courses, exams, and ultimately clinical practice.",
    "edition": "5th edition",
    "price": 79.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-06.jpeg",
    "title": "Human Embryology and Developmental Biology",
    "url": "https://www.inkling.com/store/book/human-embryology-and-developmental-biology-carlson-5th/",
  },
  Object {
    "attribution": "Kenneth Murphy, Raif Geha, and Luigi Notarangelo",
    "briefDescription": "",
    "chapterPrice": 9.99,
    "copyrightYear": 2011,
    "description": "Learn the intricacies of the human immune system with two textbooks in one. On Inkling, Janeway's Immunobiology includes Case Studies in Immunology: A Clinical Companion. Quickly access dozens of cases, view over 40 animations, test yourself with 30+ interactive figures and analyze complex processes at your own pace with 105 slidelines. With the Inkling edition, you'll be immune to bad test scores.
",
    "edition": "8th edition",
    "price": 129.99,
    "primaryCategory": "Medicine",
    "publisher": "Garland Science",
    "subCategories": Array [],
    "subtitle": "With Case Studies in Immunology, 6th edition",
    "thumbnail": "data/thumbnails/book-18.jpeg",
    "title": "Janeway’s Immunobiology",
    "url": "https://www.inkling.com/store/book/janeways-immunobiology-kenneth-murphy-8th/",
  },
  Object {
    "attribution": "Brooks et al.",
    "briefDescription": "",
    "chapterPrice": 6.99,
    "copyrightYear": 2012,
    "description": "Studying medical microbiology? Try Jawetz, Melnick & Adelberg's Medical Microbiology on Inkling. Inkling's version integrates concise insight into the role of microorganisms with interactive study tools like multiple choice quizzes with immediate answer feedback, hi-res images, intuitive navigation and a powerful search. Take the free chapter below for a spin!",
    "edition": "26th edition",
    "price": 65,
    "primaryCategory": "Medicine",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-10.jpeg",
    "title": "Jawetz, Melnick & Adelberg's Medical Microbiology",
    "url": "https://www.inkling.com/store/book/jawetz-melnick-adelbergs-medical-microbiology-26th/",
  },
  Object {
    "attribution": "Richard A. Harvey and Cynthia Nau Cornelissen",
    "briefDescription": "",
    "chapterPrice": 4.99,
    "copyrightYear": 2012,
    "description": "Master the essential concepts of microbiology with the Inkling version of this Lippincott's Illustrated Review. Ace the boards and study with interactive study tools like a powerful search, hi-res, zoomable images and note-sharing with classmates. Plus, referenced content, page numbers, and figures are cross-linked for easy navigation.",
    "edition": "3rd edition",
    "price": 59.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-07.jpeg",
    "title": "Lippincott's Illustrated Reviews: Microbiology",
    "url": "https://www.inkling.com/store/book/lippincotts-illustrated-reviews-microbiology-harvey-3rd/",
  },
  Object {
    "attribution": "Nalini Chandar and Susan Viselli",
    "briefDescription": "",
    "chapterPrice": 4.99,
    "copyrightYear": 2010,
    "description": "Want to ace the boards? Grab the Inkling version of LIR: Cell and Molecular Biology. Go step by step through complex processes with 21 Guided Tours and spin over a dozen 3D compounds on their axes with just your fingertips. At the end of each chapter, see what you've learned using USMLE®-style quizzes with right answer feedback.",
    "edition": "1st edition",
    "price": 47.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [
      "Science",
    ],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-09.jpeg",
    "title": "Lippincott’s Illustrated Reviews: Cell and Molecular Biology",
    "url": "https://www.inkling.com/store/book/cell-and-molecular-biology-chandar-viselli-1st/",
  },
  Object {
    "attribution": "Murray et al.",
    "briefDescription": "",
    "chapterPrice": 10.99,
    "copyrightYear": 2013,
    "description": "Studying medical microbiology? Try Murray’s Medical Microbiology on Inkling. Inkling’s version integrates the facts with interactive study tools for best practice learning, including 100+ slideshows, hundreds of self-test questions embedded in the text, and powerful search. Curious yet? Take the free chapter below for a spin!",
    "edition": "7th edition",
    "price": 86.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-05.jpeg",
    "title": "Medical Microbiology",
    "url": "https://www.inkling.com/store/book/medical-microbiology-murray-rosenthal-pfaller-7th/",
  },
  Object {
    "attribution": "David Greenwood",
    "briefDescription": "",
    "chapterPrice": 11.99,
    "copyrightYear": 2012,
    "description": "Building on huge previous success and now fully revised, Medical Microbiology is explicitly geared to clinical practice. It is an ideal textbook for medical & biomedical students and specialist trainees. It will also prove invaluable to medical laboratory scientists and all other busy professionals who require a clear, current and most trusted guide to this fascinating field. This version integrates the facts with interactive study tools for best practice learning, intuitive navigation and powerful search.",
    "edition": "18th edition",
    "price": 94.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-04.jpeg",
    "title": "Medical Microbiology",
    "url": "https://www.inkling.com/store/book/medical-microbiology-greenwood-slack-barer-irving-18th/",
  },
  Object {
    "attribution": "Marjorie Kelly Cowan",
    "briefDescription": "",
    "chapterPrice": 7.99,
    "copyrightYear": 2011,
    "description": "Get to know the smallest of organisms up close and personal with Cowan’s Microbiology on Inkling. Spin dozens of 3D molecules on their axes, watch more than 80 animations of various processes and explore more than 100 enhanced figures, all exclusive to Inkling.",
    "edition": "3rd edition",
    "price": 129.99,
    "primaryCategory": "Science",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "A Systems Approach",
    "thumbnail": "data/thumbnails/book-10.jpeg",
    "title": "Microbiology",
    "url": "https://www.inkling.com/store/book/microbiology-marjorie-kelly-cowan-3rd/",
  },
  Object {
    "attribution": "Richard Goering",
    "briefDescription": "",
    "chapterPrice": 19.99,
    "copyrightYear": 2013,
    "description": "Mims’ Microbiology makes it easy for you to learn the microbiology and basic immunology concepts you need to know for your courses and USMLE. Using a clinically relevant, systems-based approach, Mims’ Microbiology makes this complex subject simple to understand and remember. In addition, this version integrates the facts with interactive study tools for best practice learning, such as note-sharing with friends, intuitive navigation and powerful search.",
    "edition": "5th edition",
    "price": 79.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "with Student Consult",
    "thumbnail": "data/thumbnails/book-11.jpeg",
    "title": "Mims' Medical Microbiology",
    "url": "https://www.inkling.com/store/book/mims-medical-microbiology-goering-dockrell-zuckerman-roitt-chiodini-5th/",
  },
  Object {
    "attribution": "Eric J. Hall and Amato Giaccia ",
    "briefDescription": "",
    "chapterPrice": 9.99,
    "copyrightYear": 2011,
    "description": "Master the concepts of diagnostic radiology and radiation oncology with Hall & Giaccia's Radiobiology for the Radiologist on Inkling. Zoom in on hi-res images, bookmark key concepts and diagrams, and search the entire text, web, and your notes to instantly find what you need! Plus, share notes with your friends and colleagues in the margins. Their responses will show up in the same thread.",
    "edition": "7th edition",
    "price": 137.99,
    "primaryCategory": "Medicine",
    "publisher": "Lippincott Williams & Wilkins",
    "subCategories": Array [
      "Medicine Professional",
    ],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-06.jpeg",
    "title": "Radiobiology for the Radiologist",
    "url": "https://www.inkling.com/store/book/radiobiology-for-the-radiologist-hall-giaccia-7th/",
  },
  Object {
    "attribution": "Ken Rosenthal",
    "briefDescription": "",
    "chapterPrice": 2.99,
    "copyrightYear": 2011,
    "description": "Rapid Review Microbiology and Immunology, 3rd Edition, by Drs. Ken S. Rosenthal and Michael J. Tan, a volume in the highly rated Rapid Review Series edited by Dr. Ed Goljan, provides a concise USMLE review, featuring outline format and images, tables and high-yield margin notes. Also includes 50 high-yield USMLE-style questions throughout.  This version integrates the facts with interactive study tools for best practice learning, such as note-sharing with friends, intuitive navigation and powerful search.",
    "edition": "3rd edition",
    "price": 42.99,
    "primaryCategory": "Medicine",
    "publisher": "Elsevier",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-11.jpeg",
    "title": "Rapid Review Microbiology and Immunology",
    "url": "https://www.inkling.com/store/book/rapid-review-microbiology-and-immunology-rosenthal-tan-3rd/",
  },
  Object {
    "attribution": "Warren Levinson",
    "briefDescription": "",
    "chapterPrice": 6.99,
    "copyrightYear": 2012,
    "description": "Want to ace the boards? Grab the Inkling version of Levinson's Review of Medical Microbiology and Immunology. Pop quiz yourself with more than 1000 USMLE®-style questions at the end of each chapter and dozens of tap-to-define key terms. Plus, get an edge with 50 Q&A Podclasses in which an expert reviews USMLE® questions, the answer choices, and the rationale behind each.",
    "edition": "12th edition",
    "price": 54.99,
    "primaryCategory": "Medicine",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-10.jpeg",
    "title": "Review of Medical Microbiology and Immunology",
    "url": "https://www.inkling.com/store/book/review-medical-microbiology-immunology-levinson-12/",
  },
  Object {
    "attribution": "Ryan et al.",
    "briefDescription": "",
    "chapterPrice": 9.99,
    "copyrightYear": 2010,
    "description": "Master the concepts of medical microbiology with the Inkling version of the popular Sherris text. Quickly find what you need with a powerful search, zoom in on hundreds of hi-res images, and use cross-links for easy navigation. It's best practice learning at its best. Try the free chapter below!",
    "edition": "5th edition",
    "price": 93,
    "primaryCategory": "Medicine",
    "publisher": "McGraw-Hill",
    "subCategories": Array [],
    "subtitle": "",
    "thumbnail": "data/thumbnails/book-13.jpeg",
    "title": "Sherris Medical Microbiology",
    "url": "https://www.inkling.com/store/book/sherris-medical-microbiology-kenneth-ryan-5th/",
  },
]
`;
